Chatify/chat/api.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from .serializers import UserSerializer
+from .models import User
+
+
+class RegistrationApi(APIView):
+    def get(self, request):
+        queryset = User.objects.all()
+        serializer = UserSerializer(queryset, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
Why returning all users in get method if you are registering user it should have only post method